python/lsst/pipe/tasks/subtractBrightStars.py

```diff
@@ -364,6 +372,32 @@ def run(
 
         return subtractorExp, invImages, badStamps
 
+    def detInRegions(self, inputs):
+        """Determine whether the input exposure's detector is in the region(s)
+        where the extended PSF model(s) is(are) available.
+
+        Parameters
+        ----------
+        inputs : `dict`
+            Dictionary containing the inputs to the task, including
+            `inputExposure` and `inputExtendedPsf`.
+
+        Returns
+        -------
+        `bool`
+            True if the detector is in the region(s) where the extended PSF
+            model(s) is(are) available, False otherwise.
+        """
+        availableDets = [
+            det
+            for detList in inputs["inputExtendedPsf"].detectors_focal_plane_regions.values()
+            for det in detList.detectors
+        ]
+        if inputs["inputExposure"].detector.getId() in availableDets:
```

Comment:
I'm a fan of full words where possible, and not using storage types in names. I think that helps readability in the future for other developers contributing to this package and maintaining it.

So here for example, `availableDetectors` for `availableDets`, `detector` for `det` and `detectors` for `detList`. However, this is not a major comment, and I'll defer to you if you feel the way you have it now is more legible in general.